Hummingbird.docc/Articles/RequestContexts.md

```diff
@@ -116,6 +120,30 @@ router.get("ip") { _, context in
 }
 ```
 
+### RequestContext transformation
+
+The `RequestContext` can be transformed for the routes in a route group. The `RequestContext` you are converting to needs to conform to ``ChildRequestContext``. This requires a parent context ie the `RequestContext` you are converting from and a ``ChildRequestContext/init(context:)`` function to perform the conversion.
+
+```swift
+struct MyNewRequestContext: ChildRequestContext {
+    typealias ParentContext = MyRequestContext
+    init(context: ParentContext) throws {
+        self.coreContext = context.coreContext
+        ...
+    }
+}
+```
+Once you have defined how to perform the transform from your original `RequestContext` the conversion is added as follows
+
+```swift
+let app = Application(context: MyRequestContext.self)
```

Comment:
You seem to intend making a `let router = Router(MyRequestContext.self)`

Reply:
whoops. Fixed